build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
         sample_app_version = '1.0'
         sample_app_code_version = 1
 
-        omise_sdk_version = '4.4.2'
-        omise_sdk_code_version = 34
+        omise_sdk_version = '4.4.3'
```

Comment:
since it's a feature, can we do `4.5.0` instead? to follow semantic versioning https://semver.org/

Reply:
otherwise our merchants setup may do an auto update as they will think it's a patch/bug fix.

Reply:
Thanks! Fixed here instead -> https://github.com/omise/omise-android/pull/241